aioxmpp/entitycaps/service.py

```diff
@@ -219,18 +221,42 @@ class EntityCapsService(aioxmpp.service.Service):
 
     .. signal:: on_ver_changed
 
-       The signal emits whenever the ``ver`` of the local client changes. This
-       happens when the set of features or identities announced in the
-       :class:`.DiscoServer` changes.
+       The signal emits whenever the Capability Hashset of the local client
+       changes. This happens when the set of features or identities announced
+       in the :class:`.DiscoServer` changes.
 
     .. autoattribute:: cache
 
+    .. autoattribute:: xep115_support
+
+    .. autoattribute:: xep390_support
+
+    .. attribute:: update_delay
+        :annotation: = 0.1
+
+        Number of seconds (can be a float) by which the update of the
+        Capability Hash Set is delayed.
+
+        This is used to rate-limit the generation of caps-related presences. It
+        can be set to 0 ot effectively remove that rate-limit if rate-limiting
+        is applied elsewhere, but using a rate-limit is encouraged.
+
+        The Capability Hash Set is updated `update_delay` seconds after the
+        most recent change to the features/identities set on the
+        :class:`aioxmpp.DiscoServer`. That means if you those are changed in
+        intervals of ``0.9*update_delay``, the Capability Hash Set is never
```

Comment:
Agreed, I’ll get rid of that rate limiting for other reasons, too. It will be the job of the consumer of on_ver_changed to ensure that proper outbound rate-limiting is applied.